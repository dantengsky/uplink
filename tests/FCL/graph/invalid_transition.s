transition initial -> terminal;

@initial
init() {
  transitionTo(:bad);
  terminate("bye");
}
