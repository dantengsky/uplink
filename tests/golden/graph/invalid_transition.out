We have the following graph errors:
Cannot transition from state: 'initial' to 'bad' in method 'init'.